org.eclipse.lsp4xml/src/main/java/org/eclipse/lsp4xml/commons/capabilities/StandardCommands.java

```diff
@@ -0,0 +1,33 @@
+/*******************************************************************************
+* Copyright (c) 2019 Red Hat Inc. and others.
+* All rights reserved. This program and the accompanying materials
+* which accompanies this distribution, and is available at
+* http://www.eclipse.org/legal/epl-v20.html
+*
+* Contributors:
+*     Red Hat Inc. - initial API and implementation
+*******************************************************************************/
+package org.eclipse.lsp4xml.commons.capabilities;
+
+/**
+ * Commonly used commands
+ * 
+ * @author Angelo ZERR
+ *
+ */
+public class StandardCommands {
+
+	private StandardCommands() {
+	}
+
+	/**
+	 * Show references
+	 */
+	public static final String SHOW_REFERENCES = "lsp.show.references";

```

Comment:
there's no standard command yet, there's not even a discussion on standard commands. `lsp.*` would imply a server side command. I'd rather prefer we stick with `xml.show.*` commands for the time being

Reply:
@fbricon updated and I move this class (ClientCommands) in the `org.eclipse.lsp4xml.settings.capabilities.codelens` since now the command is specific to xml